hack/deploy_traffic_flow_tests.sh

```diff
@@ -5,6 +5,14 @@ export KUBECONFIG=/root/kubeconfig.ocpcluster
 nodes=$(oc get nodes)
 export worker=$(echo "$nodes" | grep -oP '^worker-[^\s]*')
 
+
+# wa for https://issues.redhat.com/browse/IIC-364
+make undeploy
+podman push $(hostname):5000/intel_vsp:dev # Intel vsp is not multiarch, make sure intel_vsp:dev is x86_64
+make local-deploy
+oc create -f examples/host.yaml
+sleep 15 # Give times for Intel VSP to configure ip on <ipu-netdev>d3
```

Comment:
How difficult would it be to check for an ip on <ipu-netdev>d3  instead of sleeping?

Reply:
this is a wa, not worth it imo to remove the sleep